erc20-bridge-token/test/BridgeToken.js

```diff
@@ -161,11 +164,13 @@ await createEmptyToken(nearTokenId, BridgeTokenFactory, BridgeTokenInstance)
 
     const lockResultProof = metadataProof;
     lockResultProof.outcome_proof.outcome.status.SuccessValue = serialize(SCHEMA, 'LockResult', {
+      prefix: RESULT_PREFIX_LOCK,
       token: nearTokenId,
       amount: 100,
       recipient: ethers.utils.arrayify(adminAccount.address),
     }).toString('base64');
     lockResultProof.outcome_proof.outcome.receipt_ids[0] = 'D'.repeat(44);
+    await BridgeTokenFactory.setTokenWhitelistMode(nearTokenId, 2);
```

Comment:
It's better to use constants for whitelisting mode in tests too. But this could be added later